Makefile

```
@@ -4,7 +4,7 @@
 
 # install solc version
 # example to install other versions: `make solc 0_8_14`
-SOLC_VERSION := 0_8_14
+SOLC_VERSION := 0_8_13
```

Probably should keep the latest available version. Since the pragma now uses the carret, it will work the same.

```suggestion
SOLC_VERSION := 0_8_16
```

---

Actually, I don't think this will have the desired effect.

You should probably include Solidity version in [`foundry.toml`](https://github.com/clio-finance/mip21-rwa-registry/blob/54a734d8344f01fe25e1503530c333595420aea8/foundry.toml)

```diff
 [profile.default]
 // ...
+solc_version = '0.8.16'
```

---

as discussed my machine is having issues with 0.8.15 and 0.8.16 - likely chip-related (apple silicon) but 0.8.14 works.